### PR TITLE
Update examples to 0.11.1

### PR DESCRIPTION
**Pull request by ThunderDeliverer**
The examples, comments, scripts and documentation were updated to reflect the changes introduced since the @rmrk-team/evm-contracts was last updated in the repository and the 0.11.1 version.

### CODE REVIEW

File: contracts/NestingMultiResource/README.md [hunk 1]
```diff
@@ -45,14 +45,28 @@ The `constructor` in this case accepts no arguments as all of the arguments requ
 - `1000`: represents the `maxSupply_` argument and sets the maximum amount of tokens in the collection
 - `100_000_000`: represents the `pricePerMint_` argument and sets the price of minting one token in wei or the lowest
 denomination of the native currency of the EVM to which the smart contract is deployed to
-- `ipfs://meta`: represents the `collectionMetadata_` argument and sets the base URI of the collection metadata
+- `ipfs://meta`: represents the `collectionMetadata_` argument and sets the URI of the collection metadata
+- `ipfs://tokenMeta`: represents the `tokenURI_` argument and sets the base URI of the token metadata
+- `msg.sender`: represents the `royaltyRecipient` argument and sets the address of the recipient of royalty fees
+- `10`: represents the `royaltyPercentageBps` argument and sets the value of royalty percentage in basis points
```
Comment (steven2308) [inline contracts/NestingMultiResource/README.md:51]: I would like to use a more real value here so it does not confuse users. For instance `500` and mention that it is equivalent to 5%

Comment (ThunderDeliverer) [reply contracts/NestingMultiResource/README.md:51]: Of course, I can use such value in all of the instances, or rather add it. I'd like to keep the lowest possible value mentioned here as well 🤔

File: contracts/SplitEquippable/README.md [hunk 2]
```diff
@@ -746,8 +806,8 @@ reference from the child's perspective. The resources will be added one by one.
 don't have the `equippableRefId`.
 
 Having added the resource entries, we can now add the valid parent reference IDs using the
-[`setValidParentRefd`](#setvalidparentrefid). For example if we want to add a valid reference for the left gem, we need
-to pass thee value of equippable reference ID of the left gem, parent smart contract address (in our case this is
+`setValidParentForEquippableGroup`. For example if we want to add a valid reference for the left gem, we need to pass
+thee value of equippable reference ID of the left gem, parent smart contract address (in our case this is
```
Comment (steven2308) [inline contracts/SplitEquippable/README.md:810]: Typo: `thee`

**Review by steven2308 — APPROVED** [review]
Just added 2 minor comments, let's merge after it 
Thanks!

**Pull request merged**